tidb-cloud: add description for Starter agent plan (#22877)
pingcap/docs: tidb-cloud/select-cluster-tier.md

The free plan is ideal for those who are getting started with {{{ .starter }}}. It provides developers and small teams with the following essential features:

> - **No cost**: This plan is completely free within the [free quota](https://www.pingcap.com/tidb-cloud-starter-pricing-details/). No credit card is required to get started.

Reviewer: ```suggestion
- **No cost**: This plan is free when usage is within the [free quota](https://www.pingcap.com/tidb-cloud-starter-pricing-details/). No credit card is required to get started.

```

- **Storage**: Provides an initial 5 GiB of row-based storage and 5 GiB of columnar storage.
- **Request Units**: Includes 50 million [Request Units (RUs)](/tidb-cloud/tidb-cloud-glossary.md#request-unit-ru) for database operations.

> **Note:**
>

> > - To better support agent platform builders who need to create a large number of lightweight instances for their end users, TiDB Cloud offers the **Agent Plan**. With this plan, each TiDB Cloud organization can create up to 100 free {{{ .starter }}} instances. To learn more and apply for the Agent Plan, fill out the [application form](https://www.pingcap.com/agent-plan/).

Author: Note for reviewers:

The URL in this line is to be finalized. I'll update it as needed.

The free plan is ideal for those who are getting started with {{{ .starter }}}. It provides developers and small teams with the following essential features:

- **No cost**: This plan is free when usage is within the [free quota](https://www.pingcap.com/tidb-cloud-starter-pricing-details/). No credit card is required to get started.

- **Storage**: Provides an initial 5 GiB of row-based storage and 5 GiB of columnar storage.
- **Request Units**: Includes 50 million [Request Units (RUs)](/tidb-cloud/tidb-cloud-glossary.md#request-unit-ru) for database operations.

> **Note:**
>

> > - To better support agent platform builders who need to create a large number of lightweight instances for their end users, TiDB Cloud offers the **Agent Plan**. With this plan, each TiDB Cloud organization can create up to 100 free {{{ .starter }}} instances. To learn more and apply for the Agent Plan, fill out the [application form](https://www.pingcap.com/agent-plan/).

Author: ```suggestion
> - To better support agent platform builders who need to create a large number of lightweight instances for their end users, TiDB Cloud offers the **Agent Plan**. With this plan, each TiDB Cloud organization can create up to 100 free {{{ .starter }}} instances. This limit applies to the total number of instances and [branches](/tidb-cloud/branch-overview.md) (branches are counted as instances). To learn more and apply for the Agent Plan, fill out the [application form](https://www.pingcap.com/agent-plan/).
```

> > - You will receive an email notification once your application for the Agent Plan is approved. You will also be notified via email and the TiDB Cloud console when the number of free {{{ .starter }}} instances in your organization reaches 80 and 100.

Author: ```suggestion
> - You will receive an email notification once your application for the Agent Plan is approved. You will also be notified via email and the TiDB Cloud console when the total number of free {{{ .starter }}} instances (including branches) in your organization reaches 80 and 100.
```

To learn more about the RU consumption of different resources (including read, write, SQL CPU, and network egress), the pricing details, and the throttled information, see [{{{ .starter }}} Pricing Details](https://www.pingcap.com/tidb-cloud-starter-pricing-details/).

### Instance Capacity Plan

> To better support agent platform builders who need to create a large number of lightweight instances for their end users, TiDB Cloud offers the **Instance Capacity Plan**. With this plan, each TiDB Cloud organization can create up to 100 free {{{ .starter }}} instances in total, with each [branch](/tidb-cloud/branch-overview.md) counted as a separate instance. To learn more and apply for the Instance Capacity Plan, fill out the [application form](https://www.pingcap.com/agent-plan/).

Author: > [!NOTE]
>
> Remember to update `https://www.pingcap.com/agent-plan/` to the accurate one when it's available.

Reviewer 2: Use `https://www.pingcap.com/programs/agentic-ai-instance-capacity`

> To better support agent platform builders who need to create a large number of lightweight instances for their end users, TiDB Cloud offers the **Instance Capacity Plan**. With this plan, each TiDB Cloud organization can create up to 100 free {{{ .starter }}} instances in total, with each [branch](/tidb-cloud/branch-overview.md) counted as a separate instance. To learn more and apply for the Instance Capacity Plan, fill out the [application form](https://www.pingcap.com/agent-plan/).

Reviewer: ```suggestion
To better support agent platform builders who need to create a large number of lightweight instances for their end users, TiDB Cloud offers the **Instance Capacity Plan**. With this plan, each TiDB Cloud organization can create up to 100 free {{{ .starter }}} instances in total, with each [branch](/tidb-cloud/branch-overview.md) counted as a separate instance. To learn more and apply for the Instance Capacity Plan, fill out the [application form](https://www.pingcap.com/programs/agentic-ai-instance-capacity).

```

To learn more about the RU consumption of different resources (including read, write, SQL CPU, and network egress), the pricing details, and the throttled information, see [{{{ .starter }}} Pricing Details](https://www.pingcap.com/tidb-cloud-starter-pricing-details/).

### Instance Capacity Plan

> To better support agent platform builders who need to create a large number of lightweight instances for their end users, TiDB Cloud offers the **Instance Capacity Plan**. With this plan, each TiDB Cloud organization can create up to 100 free {{{ .starter }}} instances in total, with each [branch](/tidb-cloud/branch-overview.md) counted as a separate instance. To learn more and apply for the Instance Capacity Plan, fill out the [application form](https://www.pingcap.com/programs/agentic-ai-instance-capacity).

Reviewer: ```suggestion
For each organization in TiDB Cloud, you can create a maximum of five [branches](/tidb-cloud/branch-overview.md) across all {{{ .starter }}} and Essential instances by default.

For each paid organization in TiDB Cloud, you can create up to 100 {{{ .starter }}} instances and branches in total, with each branch counted as a separate instance.

For paid organizations building agent platforms or other services that require a large number of instances and branches, TiDB Cloud offers the **Instance Capacity Plan**. With this plan, each paid TiDB Cloud organization can create more than five branches and is not subject to the limit of 100 {{{ .starter }}} instances and branches. To learn more and apply for the Instance Capacity Plan, fill out the [application form](https://www.pingcap.com/programs/agentic-ai-instance-capacity).
```

> You will receive an email notification once your application for the TiDB Cloud Instance Capacity Plan is approved. You will also be notified via email and the TiDB Cloud console when the total number of free {{{ .starter }}} instances (including branches) in your organization reaches 80 and 100.

Reviewer: ```suggestion
You will receive an email notification once your application for the TiDB Cloud Instance Capacity Plan is approved. 
```

Reviewer: Reason: the notification function will be available in a future release

To learn more about the RU consumption of different resources (including read, write, SQL CPU, and network egress), the pricing details, and the throttled information, see [{{{ .starter }}} Pricing Details](https://www.pingcap.com/tidb-cloud-starter-pricing-details/).

### Instance Capacity Plan

For each organization in TiDB Cloud, you can create a maximum of five [branches](/tidb-cloud/branch-overview.md) across all {{{ .starter }}} and Essential instances by default.

For each paid organization in TiDB Cloud, you can create up to 100 {{{ .starter }}} instances and branches in total, with each branch counted as a separate instance.

For paid organizations building agent platforms or other services that require a large number of instances and branches, TiDB Cloud offers the **Instance Capacity Plan**. With this plan, each paid TiDB Cloud organization can create more than five branches and is not subject to the limit of 100 {{{ .starter }}} instances and branches. To learn more and apply for the Instance Capacity Plan, fill out the [application form](https://www.pingcap.com/programs/agentic-ai-instance-capacity).

> 

Reviewer 3: ```suggestion

```
